### PR TITLE
Added new Ironpython method to documentation

### PR DESCRIPTION
**Pull request by maxcapodi78**

### CODE REVIEW

File: doc/source/API/ClientServer.rst [hunk 1]
```diff
@@ -120,3 +120,36 @@ As an alternative, the user can upload the script to run to the server and run i
     upload(local_script, remote_script, "servername")
     ansysem = "/path/to/AnsysEMxxx/Linux64"
     my_client.root.run_script(remote_script, ansysem_path=ansysem)
+
+
+CPython on Linux with client-server
```
Comment (PipKat) [inline doc/source/API/ClientServer.rst:125]:
```suggestion
CPython on Linux with Client-Server
```

File: doc/source/API/ClientServer.rst [hunk 2]
```diff
@@ -120,3 +120,36 @@ As an alternative, the user can upload the script to run to the server and run i
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Pyaedt can also be launched on a linux machine, locally to bypass current Ironpython limits. To do this,
+the steps are:
```
Comment (PipKat) [inline doc/source/API/ClientServer.rst:129]:
```suggestion
To bypass current IronPython limits, you can launch PyAEDT on a Linux machine:
```

File: doc/source/API/ClientServer.rst [hunk 3]
```diff
@@ -120,3 +120,36 @@ As an alternative, the user can upload the script to run to the server and run i
...
+
+1. Install pyaedt on linux using pip  (minimum version 0.4.23)
```
Comment (PipKat) [inline doc/source/API/ClientServer.rst:131]:
```suggestion
1. Using pip, install PyAEDT 0.4.23 or later on a Linux machine. 
```

File: doc/source/API/ClientServer.rst [hunk 4]
```diff
@@ -120,3 +120,36 @@ As an alternative, the user can upload the script to run to the server and run i
...
+2. Launche CPython and run pyaedt
```
Comment (PipKat) [inline doc/source/API/ClientServer.rst:132]:
```suggestion
2. Launche CPython and run PyAEDT.
```

File: doc/source/API/ClientServer.rst [hunk 5]
```diff
@@ -120,3 +120,36 @@ As an alternative, the user can upload the script to run to the server and run i
...
+
+.. code:: python
+
+    # Launch the latest installed version of AEDT in graphical mode.
```
Comment (PipKat) [inline doc/source/API/ClientServer.rst:136]:
```suggestion
    # Launch the latest installed version of PyAEDT in graphical mode.
```

File: doc/source/API/ClientServer.rst [hunk 6]
```diff
@@ -120,3 +120,36 @@ As an alternative, the user can upload the script to run to the server and run i
...
+
+    from pyaedt.common_rpc import launch_ironpython_server
+    client = launch_ironpython_server(ansysem_path="/path/to/ansys/executable/folder", non_graphical=True, port=18000)
+    hfss = client.root.hfss()
+    # put your code here
+
+3. In case the method returns a list or dictionary there may be issue in handling with CPython.
+   In this case use the following method to workaround the problem.
```
Comment (PipKat) [inline doc/source/API/ClientServer.rst:144]:
```suggestion
3. If the method returns a list or dictionary, use the following method to work around a handling
    issue in CPython:
```

Comment (PipKat) [inline doc/source/API/ClientServer.rst:144]:
```suggestion
3. If the method returns a list or dictionary, use the following method to work around an issue with CPython handling:
```

**Comment by PipKat** [comment]
@MaxJPRey Left some editorial review suggestions.

**Review by MaxJPRey — APPROVED** [review]

**Pull request merged**